f2/src/forms/WhatHappenedForm.js

```diff
@@ -2,23 +2,14 @@
 import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { Trans } from '@lingui/macro'
-import { Form } from 'react-final-form'
-import { Field } from '../components/Field'
 import { TextArea } from '../components/text-area'
```

Comment:
Please use the textarea component from f2\src\components\formik\textArea\index.js

Reply:
Fixed